src/main/java/datagateways/DataManager.java

```diff
@@ -0,0 +1,24 @@
+package datagateways;
```

Comment:
The package naming convention is a little bit wrong
I think it should be named `dataGateways` instead of all lower case

Reply:
We choose the name convention with all lower cases for the package name. Other packages' name will be change later